ravenframework/utils/xmlUtils.py

```diff
@@ -15,8 +15,6 @@
 Tools used to format, edit, and print XML in a RAVEN-like way
 talbpaul, 2016-05
 """
-
-from __future__ import division, print_function, unicode_literals, absolute_import
```

Comment:
This line should not be removed because on some machines, xmlUtils is used in python 2 when installing plugins, and this causes the plugins to not properly be tested.

Reply:
readied